crm_facebook_leads/migrations/15.0.1.0.0/pre-migration.py

```diff
@@ -1,17 +0,0 @@
-from openupgradelib import openupgrade
```

Comment:
Since openupgradelib is not used anymore, it may be removed from requirements

Reply:
Ready!